src/substrait/textplan/Any.h

```diff
@@ -9,7 +9,7 @@
 namespace io::substrait::textplan {
 
 template <class ValueType>
-inline ValueType any_cast(const std::any& value, const char* file, int line) {
+inline ValueType AnyCast(const std::any& value, const char* file, int line) {
```

Comment:
Since this function wraps std::any_cast it seemed appropriate to use the same style.